services/web/server/src/simcore_service_webserver/projects/_projects_db.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+import sqlalchemy as sa
+from aiohttp import web
+from models_library.projects import ProjectID
+from simcore_postgres_database.utils_repos import transaction_context
+from simcore_postgres_database.webserver_models import projects
+from sqlalchemy.ext.asyncio import AsyncConnection
+
+from ..db.plugin import get_asyncpg_engine
+from .exceptions import ProjectNotFoundError
+from .models import ProjectDB
+
+_logger = logging.getLogger(__name__)
+
+
+# NOTE: MD: I intentionally didn't include the workbench. There is a special interface
+# for the workbench, and at some point, this column should be removed from the table.
+# The same holds true for access_rights/ui/classifiers/quality, but we have decided to proceed step by step.
```

Comment:
maybe also create an issue to summarise what needs to be done here. Otherwise this TODO comment will be forgotten.
And also link the issue here